src/include/cephfs/libcephfs.h

```diff
@@ -44,6 +44,14 @@ extern "C" {
 #define LIBCEPHFS_VERSION(maj, min, extra) ((maj << 16) + (min << 8) + extra)
 #define LIBCEPHFS_VERSION_CODE LIBCEPHFS_VERSION(LIBCEPHFS_VER_MAJOR, LIBCEPHFS_VER_MINOR, LIBCEPHFS_VER_EXTRA)
 
+#if __GNUC__ >= 4
```

Comment:
i think we can assume that we are using GCC > 4 now.

Reply:
Yes, I guess so too. But this is copied from `src/include/rbd/librbd.h`. Could that (still?) be there for compatibility with non-gcc compilers?